addons/hr_timesheet/report/report_timesheet_templates.xml

```diff
@@ -72,21 +72,25 @@
                         <div class="oe_structure"/>
                         <div class="row mt8">
                             <div class="col-12">
-                                <t t-if="doc.allow_timesheets and doc.timesheet_ids">
-                                    <h1 class="my-4">
-                                        <t t-if="not show_record">
-                                            <t t-out="title"/>: <span t-field="doc.name"/>
-                                        </t>
-                                    </h1>
-                                    <h2>
-                                        <span>Timesheets
-                                            <t t-if="show_record">
-                                                for the <t t-out="doc.name"/> <t t-out="title"/>
+                                <t t-set="tasks" t-value="doc.timesheet_ids and not show_task and doc.child_ids and doc + doc.child_ids or doc"/>
+                                <t t-foreach="tasks" t-as="doc">
+                                    <t t-if="doc.allow_timesheets and doc.timesheet_ids">
+                                        <h1 class="my-4">
+                                            <t t-if="not show_record">
+                                                <t t-out="title"/>: <span t-field="doc.name"/>
                                             </t>
-                                        </span>
-                                    </h2>
-                                    <t t-set='lines' t-value='doc.timesheet_ids'/>
-                                    <t t-call="hr_timesheet.timesheet_table"/>
+                                        </h1>
+                                        <h2>
+                                            <span>Timesheets
+                                                <t t-if="show_record">
+                                                for the <t t-out="doc.name"/> <t t-if="not show_task and doc.parent_id" t-out="'Sub-tasks'"/>
```

Comment:
Using t-out will not allow for the text to be translated, create a full tag and put the text in it
```suggestion
                                                for the <t t-out="doc.name"/> <t t-if="not show_task and doc.parent_id">Sub-tasks</t>
```

Reply:
Hello @MissingNoShiny 
I have done the necessary changes.
Thank You :)